docs/source/04-06-slots-and-plugins.md

```diff
@@ -0,0 +1,169 @@
+    "talk-plugin-sort-newest",
+    "talk-plugin-sort-oldest",
+    "talk-plugin-viewing-options",
+    "talk-plugin-profile-settings"
+  ]
+}
+```
+
+Let's only focus on the plugins which are listed under `client` - these are the plugins that use *slots* to inject certain functionality into the Talk UI.
+
+For example, if we look at the Respect plugin (`talk-plugin-respect`), we can see its `client/index.js` looks like this:
+
+
+```js
+import RespectButton from './RespectButton';
+import translations from './translations.yml';
+
+export default {
+  translations,
+  slots: {
+    commentReactions: [RespectButton],
+  },
+};
+
+```
+
+Inside the `slots` property, we specify which **slots** the plugin will use. Above we are saying that the `RespectButton` component is being injected into the slot `commentReactions`.
+
+Slots can receive an Array of components, so we can use one plugin or many for one slot.
+
+### Anatomy of the Slot Component
+
+In Talk core, we have 32 slots available for us to use. The component `Slot` has a `fill` property where we establish the name of the slot. It looks like this:
+
+
+```js
+<Slot 
+  fill="commentReactions"
+	{...props}
+/>
+```
+
+You won't have to use this to build plugins, but it's helpful to find where to embed your plugin.
+
+### Slot List
+
+* `adminCommentDetailArea`
+* `adminCommentMoreDetails`
+* `adminCommentLabels`
+* `adminModerationSettings`
+* `adminStreamSettings`
+* `adminTechSettings`
+* `adminCommentInfoBar`
+* `adminCommentContent`
+* `adminSideActions`
+* `adminModeration`
+* `adminModerationIndicator`
+
+* `commentInputDetailArea`
+* `commentAvatar`
+* `commentAuthorName`
+* `commentAuthorTags`
+* `commentTimestamp`
+* `commentInfoBar`
+* `commentContent`
+* `commentReactions`
+* `commentActions`
+* `commentInputArea`
+
+* `draftArea`
+* `streamSettings`
+* `historyCommentTimestamp`
+* `profileSections`
+* `embed`
+* `stream`
+* `streamFilter`
+* `streamQuestionArea`
+* `login`
+* `userProfile`
+* `userDetailCommentContent`
+
+### Where should I insert my plugin?
+
+The first thing we should consider is what components will be affected by our plugin's functionality. For example, if we want to add functionality to all the comments that are rendered in a total list of comments, we would use the component `Comment`.
+
+The slots that are able to add functionality to comments start with `comment`, like `commentContent`, or `commentReactions`, as you can see above.
+
+### Disabling plugins via `plugins_config`
+
+Typically, you will manage plugins via your `plugins.json` file. If you want to remove a plugin, you would simply delete it there. However, we can also do this directly with the `plugins_config`.
+
+Let's look at our example article, `views/article.ejs`. Here we see can we have the Talk embed, and within the embed, we can also send a configuration object. To disable a plugin visually, we can pass `true` to the property `disable_components`. Like so:
```

Comment:
I guess you meant: `Here  we have the Talk embed, ...`

Reply:
"Here we can see we have the Talk embed..." is also correct - is it not clear?

Reply:
Is that correct English? I thought `that` would be required, e.g. `Here we can see that we have the Talk embed`

Reply:
Oh lol, it's a typo, sorry.

Reply:
Fixed!